src/wp-includes/capabilities.php

```diff
@@ -801,6 +801,32 @@ function map_meta_cap( $cap, $user_id, ...$args ) {
 		case 'delete_app_password':
 			$caps = map_meta_cap( 'edit_user', $user_id, $args[0] );
 			break;
+		case 'edit_block_binding':
+			$block_editor_context = $args[0];
+			if ( isset( $block_editor_context ) && isset( $block_editor_context->post ) ) {
```

Comment:
```suggestion
			if ( isset( $block_editor_context->post ) ) {
```

that’s the equivalent but shorter